src/Nethermind/Nethermind.Network/P2P/Subprotocols/Eth/V65/Eth65ProtocolHandler.cs

```diff
@@ -99,7 +99,7 @@ protected virtual void Handle(NewPooledTransactionHashesMessage msg)
 
             AddNotifiedTransactions(msg.Hashes);
 
-            Stopwatch stopwatch = Stopwatch.StartNew();
```

Comment:
We typically don't use `var`, the same for the code above and below

Reply:
so swap var for actual type?

Reply:
Yup